src/mods/2.78/analyzer/update/bpy.props.mod.rst

```diff
@@ -0,0 +1,8 @@
+.. mod-type:: update
+
+.. module:: bpy.props
+
+.. function:: EnumProperty(items, name, description, default, options, update, get, set)
```

Comment:
You can fix this like below code.
By using this syntax, you don't need to make mod file for all versions.

```suggestion
.. function:: EnumProperty(items)
```

Reply:
Ah, I tried with no arguments at all, but didn't consider trying with just `items`, that works nicely. 👍